### PR TITLE
add/improve pokerth cleaner, ref #108

### PR DESCRIPTION
**Pull request by nodiscc**

### CODE REVIEW

File: release/pokerth.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+    
+    CleanerML  : PokerTH 1.1.1 (www.PokerTH.net)
+    Contributor: Erratalog
+    Web site   : http://erratalog.blogspot.com
+    
+-->
+<cleaner id="pokerth">
```
Comment (az0) [inline release/pokerth.xml]: Please add `os="windows"`

File: release/pokerth.xml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    BleachBit
+    Copyright (C) 2008-2015 Andrew Ziem
+    http://bleachbit.sourceforge.net
```
Comment (az0) [inline release/pokerth.xml]: The new address is preferred https://www.bleachbit.org

Comment (nodiscc) [inline release/pokerth.xml]: Done for all cleaners https://github.com/az0/cleanerml/commit/55a24dbc7fad1a001fa1b0b417ad53da093fae5c

**Comment by az0** [comment]
@nodiscc Thank you for updating the address :) I really appreciate your help with this project

**Pull request merged**